pywb/apps/cli.py

```diff
@@ -76,7 +78,8 @@ def __init__(self, args=None, default_port=8080, desc=''):
             self.extra_config['proxy'] = {
                 'coll': self.r.proxy,
                 'recording': self.r.proxy_record,
-                'enable_wombat': self.r.proxy_enable_wombat
+                'enable_wombat': self.r.proxy_enable_wombat,
+                'timestamp': self.r.proxy_timestamp,
```

Comment:
Should this be `proxy_timestamp` instead of `timestamp`?
I think this would get confusing as `timestamp` would have to meaning:
- replay datetime
- proxy datetime

Reply:
Its used by frontendapp.py as 
```python3
self.proxy_timestamp = proxy_config.get('timestamp')
```

Reply:
Maybe it should be `default_timestamp`?

The reason is in the yaml config its already in the proxy options
```
proxy:
   coll: ...
   proxy_timestamp:
```
so proxy_timestamp seems redundant..

Naming is hard :)

Reply:
#hardestproblemcompsci